webapp/channels/src/components/more_direct_channels/list/list.tsx

```diff
@@ -96,9 +96,32 @@ const List = React.forwardRef((props: Props, ref?: React.Ref<MultiSelect<OptionV
 
     let remainingText;
     if (MAX_SELECTABLE_VALUES >= props.values.length) {
-        remainingText = 'Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. ';
+        remainingText = (
+            <FormattedMessage
+                id={'multiselect.numPeopleRemaining'}
+                defaultMessage={'Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. '}
+                values={{
+                    num: MAX_SELECTABLE_VALUES - props.values.length,
+                }}
+            />
+        );
     } else {
-        remainingText = 'Use ↑↓ to browse, ↵ to select. You can\'t add more than 7 people. Please <a>create a channel</a> to include more people. ';
+        remainingText = (
+            <FormattedMessage
+                id={'multiselect.maxPeople'}
+                defaultMessage={'Use ↑↓ to browse, ↵ to select. You can\'t add more than 7 people. Please <a>create a channel</a> to include more people.'}
```

Comment:
Sorry for all the back and forth. Can you make one small change more? I don't expect we ever change the value of `MAX_SELECTABLE_VALUES`, but just in case, change the 7 here for a variable (`{num}` instead of `7`) and pass it directly the `MAX_SELECTABLE_VALUES` in the values. That way, if we ever change it, we don't need to update the translations.

Reply:
I completed the task, but I found another bug. When I try to close the direct message modal using dispatch, I can't find a way to do it; so I used props instead. However, this approach failed in testing.

Reply:
@hmhealey Any idea why dispatch may fail here?

Reply:
Because the "show" state of the MoreDirectChannels modal is in the component, not in the global store.

Reply:
I'd assume that it's because the modal's identifier is different than the one it was opened with. This way of closing the modal is actually preferable though because it's a bit simpler. The only reason you should really need to dispatch `closeModal` is if something outside of the modal can close it